packages/ember-glimmer/lib/template.js

```diff
@@ -36,14 +36,16 @@ class Wrapper {
 
 let templateId = 0;
 export default function template(json) {
+  let { meta, template } = JSON.parse(json);
```

Comment:
We wanted to avoid parsing the JSON at eval time, which this undoes...

Why do we want the meta on the template factories in addition to the instances?